manifests/templates/resourcegroup-manifest.yaml

```diff
@@ -21,288 +21,6 @@ metadata:
     control-plane: controller-manager
   name: resource-group-system
 ---
```

Comment:
Any idea why this file is in templates? It doesn't look templated....

Reply:
The image for the `resource-group-controller-manager` Deployment gets set during the build

Reply:
could that be set with kustomize? or is that what the templates dir is for?

Reply:
Yes, the templates dir is primarily for manifests that get hydrated with image names during the build process